modules/monitoring/application/views/scripts/show/components/notes.phtml

```diff
@@ -38,7 +38,7 @@ if (($navigation->isEmpty() || ! $navigation->hasRenderableItems()) && $notes ==
     <td>
         <?= $navigation->getRenderer() ?>
     <?php if ($notes !== ''): ?>
-        <p><?= $this->nl2br($this->escape($notes)) ?></p>
+        <section class="notes"><?= $this->markdown($notes) ?></section>
```

Comment:
Please use just a `div` here. We are far away from using semantic HTML 😆

Reply:
It's a `<section>` because of `#` == `<h1>` and multiple `<h1>` are only valid if separated by `<section>`.

Reply:
That makes sense.